lua/examples/policy.lua

```diff
@@ -147,12 +145,91 @@ local function lookup_simple_policy(simple_policy, pkt_info)
 	return nil
 end
 
+--[[
+The following defines the simple policies with LPM for Grantor.
+
+General format of the simple policies should be:
+	IPv4 tables: lpm table
+	IPv6 tables: lpm table
+
+Here, I assume that each group has specific capability parameters,
+including speed limit, expiration time, actions - DENY or ACCEPT, etc.
+--]]
+
+-- This file only contains an example set of Bogons IPv4 lists
+-- downloaded from http://www.team-cymru.org/Services/Bogons/fullbogons-ipv4.txt
+local bogons_ipv4_file = "lua/examples/bogons-ipv4.txt"
+local lpm = lpmlib.new_lpm(1024, 256)
+if lpm == nil then
+	return "gt: failed to initialize the IPv4 LPM table for Lua policies"
```

Comment:
Since `lpmlib.new_lpm()` is a Lua function, we don't need to check for errors; unless we want to catch an exception.

Do the same for `lpmlib.new_lpm6()` below.